core/src/server/rpc_module.rs

```diff
@@ -504,6 +504,13 @@ impl<Context> RpcModule<Context> {
 	pub fn new(ctx: Context) -> Self {
 		Self { ctx: Arc::new(ctx), methods: Default::default() }
 	}
+
+	/// Transform a module into an `RpcModule<()>` (unit context).
+	pub fn remove_context(self) -> RpcModule<()> {
+		let mut module = RpcModule::new(());
+		module.merge(self).expect("Merging with an empty module can never have conflicting method names.");
```

Comment:
```suggestion
		module.methods = self.methods;
```

Simpler, faster, and no need for `expect`.